lib/internal/per_context/primordials.js

```diff
@@ -710,5 +710,22 @@ primordials.SafeStringPrototypeSearch = (str, regexp) => {
   return match ? match.index : -1;
 };
 
+// The built-in Array#join is slower in v8 6.0
+function join(output, separator) {
```

Comment:
The default value for `separator` is `,`. Can you update your implementation? Referencing: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join